### ISSUE TITLE
Use of pyeto contravenes license

### ISSUE THREAD

**Issue by willu47** [KTH-dESA/FAO#17]
Note the terms of the [license of pyeto](https://github.com/woodcrafty/PyETo/blob/master/LICENSE.txt)

Note that generally, you shouldn't copy the source code for a dependent package. Instead install it using pip: `pip install pyeto` or include in a `requirements.txt` file in the root of the directory.

If you need to extend the pyeto library, then either fork the original repository and contribute back to pyeto, or install directly from your fork of the source code.

You can pip install directly from a git repository using `pip install git+https://github.com/user/repo.git@branch`

**Comment by camiloramirezgo** [comment]
I tried to install `pyeto` with `pip install pyeto` but it gave me this error:
```
ERROR: Could not find a version that satisfies the requirement pyeto (from versions: none)
ERROR: No matching distribution found for pyeto
``` 
I searched in the PyPI website for the `pyeto` package and apparently there is not a package called that there. Moreover, if you check the `README.md` file of the `pyeto` repository, the installation section says the following:
>TODO: complete this section

Then it seams they never uploaded this package to PyPI. 

@willu47 Is it ok if we keep using it withing our project?
